src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
```

Comment:
Its good to allow the elements inside dictate the width and height of the block.